Header header
ownage_bot/RichObject[] objects
